src/containers/my-courses/course-toolbar/CourseToolbar.tsx

```diff
@@ -30,20 +30,22 @@ import {
 } from '~/types'
 import { styles } from '~/containers/my-courses/course-toolbar/CourseToolbar.style'
 
+type CourseFormEventHandler<TEvent> = (
+  key: keyof CourseForm
+) => (event: TEvent) => void
+
+type HandleNonInputValueChange = (
+  key: keyof CourseForm,
+  value: string | ProficiencyLevelEnum[] | null
+) => void
```

Comment:
move to types and reuse as all other useForm functions

Reply:
Fixed